modules/ROOT/pages/mule-error-concept.adoc

```diff
@@ -62,30 +62,30 @@ When it comes to connectors, each connector defines its error type hierarchy con
 
 Below is listed a full list of Error Types:
 
-* `ANY`: Error type that matches all error types that may occur in an Flow, that can be handled. This does not include errors that may occur on the source. 
-** `TRANSFORMATION`: indicates an error ocurred while transforming a value.
+* `ANY`: Error type that matches all error types that may occur in an Flow and can be handled. This does not include errors that may occur on the source. 
+** `TRANSFORMATION`: indicates an error ocurred while transforming a value. This involves Mule Runtime transformation and not DataWeave transformations. For example, using the Expression Transformer. 
```

Comment:
What do you mean by expression transformer? AFAIK that's no longer part of mule 4, maybe compatibility. just say this refers to internal mule transformations.

Reply:
```suggestion
** `TRANSFORMATION`: indicates an error ocurred while transforming a value. This involves Mule Runtime internal transformations and not DataWeave transformations. 
```